src/osiris_log.erl

```diff
@@ -1811,7 +1881,8 @@ timestamp_idx_scan(Fd, Ts) ->
          <<ChunkId:64/unsigned,
            Timestamp:64/signed,
            Epoch:64/unsigned,
-           FilePos:32/unsigned>>} ->
+           FilePos:32/unsigned,
+           ?CHNK_USER:8/unsigned>>} ->
```

Comment:
Good catch, thanks.
I thought it might be kind of an optimisation when there are in the worst case let's say 100s of non-user chunks in sequence.
Removed that optimisation.